vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -103,7 +118,15 @@ def request_copy
     prov = MiqRequest.find_by_id(params[:id])
     @redirect_controller = "miq_request"
     @refresh_partial = "prov_copy"
-    @org_controller = prov.kind_of?(MiqHostProvisionRequest) ? "host" : "vm"                            #request originated for resource_type
+    case prov.resource_type
+    when "MiqHostProvisionRequest"
+      @org_controller = "host"
+    when "MiqProvisionConfiguredSystemRequest"
+      @org_controller = "configured_system"
+      @prov_id = prov.options[:src_configured_system_ids]
+    else
+      @org_controller = "vm"
+    end
```

Comment:
Instead of this case statement, can we just add an `org_controller` method to the models like this:

```diff
diff --git a/vmdb/app/models/miq_host_provision_request.rb b/vmdb/app/models/miq_host_provision_request.rb
index 1284954..53e915d 100644
--- a/vmdb/app/models/miq_host_provision_request.rb
+++ b/vmdb/app/models/miq_host_provision_request.rb
@@ -26,6 +26,10 @@ class MiqHostProvisionRequest < MiqRequest
     end
   end
 
+  def org_controller
+    "host"
+  end
+
   def requested_task_idx
     self.options[:src_host_ids]
   end
diff --git a/vmdb/app/models/miq_provision_configured_system_request.rb b/vmdb/app/models/miq_provision_configured_system_request.rb
index f67d0f9..c53456f 100644
--- a/vmdb/app/models/miq_provision_configured_system_request.rb
+++ b/vmdb/app/models/miq_provision_configured_system_request.rb
@@ -15,6 +15,10 @@ class MiqProvisionConfiguredSystemRequest < MiqRequest
     options[:src_configured_system_ids].length == 1 ? src_configured_systems.pluck(:hostname).first : "Multiple Hosts"
   end
 
+  def org_controller
+    "configured_system"
+  end
+
   def src_configured_systems
     ConfiguredSystem.where(:id => options[:src_configured_system_ids])
   end
diff --git a/vmdb/app/models/miq_request.rb b/vmdb/app/models/miq_request.rb
index fd8e93c..52e372b 100644
--- a/vmdb/app/models/miq_request.rb
+++ b/vmdb/app/models/miq_request.rb
@@ -85,6 +85,10 @@ class MiqRequest < ActiveRecord::Base
     self.class.name
   end
 
+  def org_controller
+    "vm"
+  end
+
   def initialize_attributes
     self.requester_name ||= requester.name                         if requester.kind_of?(User)
     self.requester      ||= User.find_by_name(self.requester_name) if self.requester_name.kind_of?(String)
```

Then change this code to just do `@org_controller = prov.org_controller`.